src/super_gradients/training/models/detection_models/csp_darknet53.py

```diff
@@ -27,19 +27,29 @@ def width_multiplier(original, factor, divisor: int = None):
         return math.ceil(int(original * factor) / divisor) * divisor
 
 
-def get_yolo_version_params(yolo_version: str, width_mult_factor: float, depth_mult_factor: float):
-    if yolo_version == 'v6.0':
-        struct = (3, 6, 9, 3)
+def get_yolo_version_params(yolo_version: str, yolo_type: str, width_mult_factor: float, depth_mult_factor: float):
```

Comment:
I wonder if we really need this function.
It seems like something that should be handled using yamls inheritance.
i.e. 
```
yolo_arch_params.yaml
      yolo_v5_arch_params.yaml
            yolo_v5_6_arch_params.yaml
            yolo_v5_3_arch_params.yaml
      yolo_x_arch_params.yaml
 ```
 and then use the proper arch params yaml in each training

Reply:
I think it's related to the comment below. Until we get rid of dictionaries altogether, we do need it